frontend/src/app/config.js

```diff
@@ -1,4 +1,6 @@
 export default {
   experimentsURL: '/api/experiments.json',
-  usageCountsURL: 'https://analysis-output.telemetry.mozilla.org/testpilot/data/installation-counts/latest.json'
+  usageCountsURL: 'https://analysis-output.telemetry.mozilla.org/testpilot/data/installation-counts/latest.json',
+  productionURL: 'https://testpilot.firefox.com',
+  stagingURL: 'https://testpilot.stage.mozaws.net'
```

Comment:
Maybe this is something to look at later, but something I want to do with this config file is give it a hostname-based overlay. So, like:

```javascript
const defaultConfig = {
 isDev: true,
 experimentsURL: '/api/experiments.json',
 usageCountsURL: 'https://analysis-output.telemetry.mozilla.org/testpilot/data/installation-counts/latest.json'
};

const hostConfig = {
  'testpilot.firefox.com': { isDev: false }
};

export default Object.assign({}, defaultConfig, hostConfig[location.hostname] || {});
```

Could do this now, or keep it in mind to implement later. Either way, it would be nice not to sprinkle host name checks everywhere if/when we want to do more things that differ between environments